src/main/java/ch/njol/skript/expressions/ExprAttacked.java

```diff
@@ -36,6 +37,7 @@
 import ch.njol.skript.doc.Events;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
```

Comment:
import can be removed. other than that looks good to me.

Reply:
Done! 👍